database/migrations/sqls/20240715163245-initial-db-up.sql

```diff
@@ -0,0 +1,86 @@
+DROP TABLE IF EXISTS users cascade;
+
+
+DROP TABLE IF EXISTS admins;
+
+
+DROP TABLE IF EXISTS bidder cascade;
+
+
+DROP TABLE IF EXISTS skill cascade;
+
+
+DROP TABLE IF EXISTS contractor cascade;
+
+
+DROP TABLE IF EXISTS bids cascade;
+
+
+DROP TABLE IF EXISTS tender cascade;
+
+
+DROP TABLE IF EXISTS tender_skill cascade;
+
+
+DROP TABLE IF EXISTS bidder_skill cascade;
+
+
+CREATE TABLE users (id PRIMARY KEY AUTO INCREMENT,
+                                          email VARCHAR(120) NOT NULL UNIQUE,
```

Comment:
Please fix indentation, is hard to review this file the way it's currently indented

Reply:
I have indented it now.